params/chromium-params.md

```diff
@@ -7,7 +7,7 @@ Chromium's implementation assigns the following values:
 
 | Name | Value |
 | ---- | ----- |
-| [Max pending sources per source origin][] | [1024][max pending sources per source origin value] |
+| [Max pending sources per source origin][] | [4096][max pending sources per source origin value] |
```

Comment:
Please also update the link below so that it goes to the updated Code Search line.